ochazuke/api/views.py

```diff
@@ -13,13 +13,16 @@
 from flask import request
 from flask import Response
 
+from urllib.error import HTTPError
```

Comment:
Note: This is needed because I want to catch the exception that could occur when requesting data from the dashboard. I throw a 400 when the request parameters are invalid, and this could crash the app.

I was thinking about modifying the `get_remote_data` helper and catch the exception there, but this would modify existing behavior for requesting other URLs, so I didn't do that. Happy to change that, though!

Reply:
understood.